src/GitTools.IssueTrackers/Context/IssueTrackerContext.cs

```diff
@@ -1,29 +0,0 @@
-﻿namespace GitTools.IssueTrackers
-{
-    public class IssueTrackerContext : Disposable, IIssueTrackerContext
-    {
-        public IssueTrackerContext()
```

Comment:
I might add this one back in, but without the interface/disposable. Might rename it to `IssueTrackerInfo` or something. Context seems the wrong name